test/fixtures/comments.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
```

Comment:
Since there are no tests in this repository, you can remove the test directory from your repository, including all files within it.

[OPTIONAL]